lib/iris/fileformats/nimrod.py

```diff
@@ -87,46 +89,95 @@
     "sat_space_count",
     "ducting_index",
     "elevation_angle",
+    "neighbourhood_radius",
+    "threshold_vicinity_radius",
+    "recursive_filter_alpha",
+    "threshold_fuzziness",
+    "threshold_duration_fuzziness",
+    "data_header_float32_05",
+    "data_header_float32_06",
+    "data_header_float32_07",
+    "data_header_float32_08",
+    "data_header_float32_09",
+    "data_header_float32_10",
+    "data_header_float32_11",
+    "data_header_float32_12",
+    "data_header_float32_13",
+    "data_header_float32_14",
+    "data_header_float32_15",
+    "data_header_float32_16",
+    "data_header_float32_17",
+    "data_header_float32_18",
+    "data_header_float32_19",
+    "data_header_float32_20",
+    "data_header_float32_21",
+    "data_header_float32_22",
+    "data_header_float32_23",
+    "data_header_float32_24",
+    "data_header_float32_25",
+    "data_header_float32_26",
+    "data_header_float32_27",
+    "data_header_float32_28",
+    "data_header_float32_29",
+    "data_header_float32_30",
+    "data_header_float32_31",
+    "data_header_float32_32",
 )
 
 
 # data specific header (int16) elements
```

Comment:
I think it would be useful to have the element number range in these comments. It makes it easier to cross-reference with the nimrod documentation

Reply:
Added